'''
try:
    # code that may occur exception
except:
    # code that handle exception
finally:
    # code that clean up
'''

# try:
#     a = int(input("Enter a: "))
#     b = int(input("Enter b: "))

#     print(a/b)

# except (ValueError, ZeroDivisionError) as error:
#     print(f"Error! -> {error}")

# finally:
#     print("Finishing up!")